falcon/request.py

```diff
@@ -2021,6 +2028,10 @@ def get_param_as_list(
             name (str): Parameter name, case-sensitive (e.g., 'ids').
 
         Keyword Args:
+            delimiter(str): An optional character for splitting a parameter
+                value into a list. Useful for styles like ``spaceDelimited``
+                or ``pipeDelimited``. If not provided, default list parsing
```

Comment:
It might be useful to link to the OpenAPI spec in the parentheses `(see also: ...)`.